ocaml/xapi/vgpuops.ml

```diff
@@ -129,7 +129,8 @@ let reserve_free_virtual_function ~__context vm pf =
 let add_vgpus_to_vm ~__context host vm vgpus vgpu_manual_setup =
   (* Only support a maximum of one virtual GPU per VM for now. *)
   match vgpus with
-  | [] -> ()
+  | [] ->
+    (try Db.VM.remove_from_other_config ~__context ~self:vm ~key:Xapi_globs.vgpu_pci with _ -> ());
```

Comment:
The outer parentheses are unnecessary, and so is the semicolon: this could be written
`try ... ()` instead of
`(try ... ());`
but it's not important.